### ISSUE TITLE
repo_install() should enable rhel-6-server-optional-rpms

### ISSUE THREAD

**Issue by lpramuk** [SatelliteQE/automation-tools#440]
Both {ak,repo}_install() should enable rhel-6-server-optional-rpms:
```
Error: Package: 1:livecd-tools-13.4.10-1.el6.x86_64 (Sat6-CI_CentOS_EPEL_6)
            Requires: syslinux-extlinux
  You could try using --skip-broken to work around the problem
  You could try running: rpm -Va --nofiles --nodigest
``` 
```syslinux-extlinux``` for rhel6 lives in ```optional``` repo

**Comment by lpramuk** [comment]
ak_install() doesn't have to enable optional.
AK bears information of repos to enable (optioonal should be among them)

**Comment by omaciel** [comment]
@lpramuk can this be closed since #441 was merged?

**Comment by kbidarkar** [comment]
Yes, this can be closed now.
